apps/marketplace/components/Header/Header.scss

```diff
@@ -1,98 +0,0 @@
-@import "../../main.scss";
```

Comment:
all the header styles went to custom.css, so they could be shared with other repos?

Reply:
Yes, that way if we change something in there it would change all the headers. It was a pain having to maintain css for three headers. There is also a lot of unused code in the python repos css, but didn't get to clean it up.